### PR TITLE
Planning Logging Refactor

### PR DESCRIPTION
**Pull request by icolwell-as**
Must be merged after https://github.com/Autoware-AI/core_planning/pull/36

Refactor of ROS logging for planning modules.

See original MR for more details: https://gitlab.com/astuff/autoware.ai/core_planning/-/merge_requests/78

### CODE REVIEW

File: waypoint_planner/src/astar_avoid/astar_avoid.cpp [hunk 1]
```diff
@@ -210,16 +209,41 @@ void AstarAvoid::run()
 
 bool AstarAvoid::checkInitialized()
 {
-  bool initialized = false;
-
   // check for relay mode
-  initialized = (current_pose_initialized_ && closest_waypoint_initialized_ && base_waypoints_initialized_ &&
-                 (closest_waypoint_index_ >= 0));
```
Comment (TakaHoribe) [inline waypoint_planner/src/astar_avoid/astar_avoid.cpp:217]: Why did you remove the last condition `closest_waypoint_index_ >= 0`?

**Review by TakaHoribe — COMMENTED** [review]
I confirmed this log works correctly. Please check one comment below.

File: waypoint_planner/src/astar_avoid/astar_avoid.cpp (hunk 1 above)
Comment (icolwell-as) [reply waypoint_planner/src/astar_avoid/astar_avoid.cpp:217]: Good question, I just realized this PR comes after this other one: https://github.com/Autoware-AI/core_planning/pull/36
It's removed in the other PR. I updated the PR description to review/merge #36 first.

Comment (TakaHoribe) [reply waypoint_planner/src/astar_avoid/astar_avoid.cpp:217]: Understood. Thank you.

**Review by JWhitleyWork — APPROVED** [review]

**Comment by JWhitleyWork** [comment]
@icolwell-as Once conflicts are resolved, I will merge.
